Content.Server/Nutrition/Components/ThirstComponent.cs

```diff
@@ -177,16 +184,26 @@ public void OnUpdate(float frametime)
 
             if (_currentThirstThreshold != ThirstThreshold.Dead)
                 return;
+            // --> Current Hunger is below dead threhsold
 
             if (!Owner.TryGetComponent(out IDamageableComponent? damageable))
                 return;
 
             if (!Owner.TryGetComponent(out IMobStateComponent? mobState))
                 return;
 
+            Logger.ErrorS("aa", $"frametime:{frametime}");
```

Comment:
meant to remove?

Reply:
Yes. That was when I was testing damage. My bad.